recipes/apache-airflow-providers-fab/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "apache-airflow-providers-fab" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apache_airflow_providers_fab-{{ version }}.tar.gz
+  sha256: f146e401a40cfbdf95567d249c9d39bda35ad77cdf61db28eb84c2f874f668d4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.dev0
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >=3.8
+    # skip for now to avoid circular dependency
+    # - apache-airflow >=2.9.0
+    - flask-appbuilder ==4.4.1
+    - flask-login >=0.6.2
+    - flask >=2.2,<2.3
+    - google-re2 >=1.0
+
+test:
+  imports:
+    - airflow.providers.fab
```

Comment:
Not sure if this import will work without `apache-airflow`. If not, I'll comment it out, too.

Reply:
Indeed, import fails.